src/components/panel/panel.spec.js

```diff
@@ -1115,6 +1223,95 @@ describe('$mdPanel', function() {
       mdPanelPosition = $mdPanel.newPanelPosition();
     });
 
+    describe('should update the position of an open panel', function() {
+      var xPosition, yPosition, myButton, myButtonRect;
+
+      beforeEach(function() {
+        xPosition = $mdPanel.xPosition;
+        yPosition = $mdPanel.yPosition;
+
+        myButton = '<button>myButton</button>';
+        attachToBody(myButton);
+        myButton = angular.element(document.querySelector('button'));
+        myButtonRect = myButton[0].getBoundingClientRect();
+      });
+
+      it('between two absolute positions', function() {
+        var top = '50px';
+        var left = '30px';
+        var position = $mdPanel.newPanelPosition().absolute().top(top).left(left);
```

Comment:
nit: these will be more readable if you chain them on the next lines, like the relative positioning setup below.